src/script/event/EventRepository.ts

```diff
@@ -603,7 +605,7 @@ export class EventRepository {
       return this.throwValidationError(newEvent, errorMessage);
     }
 
-    const textContentMatches = newEventData.content === originalData.content;
+    const textContentMatches = newEventData.content === (originalData as any).content;
```

Comment:
I have a followup PR that will extract all this code to a new module. This will be fixed there